test/test-network/systemd-networkd-tests.py

```diff
@@ -1284,7 +1284,11 @@ def test_bridge_property(self):
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'hairpin_mode'), '1')
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'path_cost'), '400')
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'unicast_flood'), '1')
+        if (os.path.exists('/sys/devices/virtual/net/bridge99/lower_dummy98/brport/multicast_flood')):
+          self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'mcast_flood'), '0')
```

Comment:
we indent python code by 4ch, see code around it

Reply:
thx. I update this as soon as I got some results from the ci